CHANGELOG.md

```diff
@@ -38,6 +38,9 @@ All notable changes to this project will be documented in this file.
         * Support for Python 3.6
 * ### `nidcpower` (NI-DCPower)
     * #### Added
+        * API parity with next NI-DCPower release.
```

Comment:
Is this a bad merge or an unrelated change?

Reply:
Bad merge, fixed